test/server/parse-fixture-test.js

```diff
@@ -38,8 +49,8 @@ function testFixtureParser (dir, expectedStructure, fileParser, codeParser) {
 
         return fileParser(filePath)
             .then(function (structure) {
-                expect(structure).eql(expected);
-                expect(codeParser(fileContent)).eql(expected);
+                lazyEql(structure, expected);
```

Comment:
I suggest using the current approach for this case - [removing new fields](https://github.com/DevExpress/testcafe/pull/5760/files).